UKHomeOffice/ide-controller
# COP 1976: MRZ table context updates

## Description
This branch allows the MRZ table to update with values taken from The scanned document. 

## To test
- run `npm ci && npm run dev`
- if the electron app displays blank, refresh with `Cmd + r` or `Ctrl + r` 
- navigate to the "Data read from chip and MRZ" tab
- submit the mock document data with `F1`
- Values should change from "No Data" to the mock data values. 

## Developer Checklist

\* Required
